### PR TITLE
Readme func and python ecosystem

### LINKED ISSUE #6: Function list and Python ecosystem in README
Incorporate the following in the README.md:
- a bulleted list of the functions (and datasets if applicable) that will be included in the package (this should be a 1-2 sentence description for each function/dataset)
- a paragraph describing where your packages fit into the Python ecosystem (are there any other Python packages that have the same/similar functionality? Provide links to any that do. If none exist, then clearly state this as well).

### PR DESCRIPTION
**Pull request by s-voon**
Hi Anu, I revised the README.md function to include the functions list and where our package fit within the phyton ecosystem, please review them. 
Note: function names are tentative and subjects to change based on the src python scripts

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,14 @@
 
 A package that streamline the repetitive task of regression models selection and comparison in the machine learning workflow.
 
+This package includes four main functions:
+- `fit`: Fits a clean, preprocessed training data into eight different regression models. The fuinction returns a dictonary containing four various metric scores for each model
```
Comment (AnuBanga) [inline README.md]: There are minor spelling errors
```suggestion
- `fit`: Fits a clean, preprocessed training data into eight different regression models. The function returns a dictionary containing four various metric scores for each model
```

**Review by AnuBanga — CHANGES_REQUESTED** [review]
Please find suggestions.

**Comment by s-voon** [comment]
Thanks for reviewing! Fixed the typos :)

**Review by AnuBanga — APPROVED** [review]
Reviewed

**Pull request merged**